website/Bomberjam.Website/Frontend/scripts/react/viewerWithControls.tsx

```diff
@@ -84,6 +84,9 @@ export const ViewerWithControls = (props: VisualizerProps) => {
               <button onClick={decreaseSpeed} className="btn btn-primary" disabled={!isStarted}>Slower</button>
               <button onClick={increaseSpeed} className="btn btn-primary" disabled={!isStarted}>Faster</button>
             </div>
+            <div className="btn btn-sm ml-2" style="pointer-events:none;">
+              <span>{selectedStateIdx}<span>
```

Comment:
When the string length of `selectedStateIdx` changes, the scrollbar is pushed or pulled horizontally. Either prefix with zeroes (`4` => `004`), or used a fixed width. I personnally prefer the prefix solution